test/unit/foreman/access_permissions_test.rb

```diff
@@ -10,59 +11,8 @@
 #
 # In particular, it's important that actions for AJAX requests are added to
 # an appropriate permission so views using those requests function.
-class AccessPermissionsTest < ActiveSupport::TestCase
-  MAY_SKIP_REQUIRE_LOGIN = [
```

Comment:
Correct, but it does allow me to write my own test and call these constants and add my own routes, such as in the commit i mentioned:  https://github.com/Katello/katello/pull/6403/files#diff-0997358e45358776de1e730a5e19cf4bR45

If i skip the test completely (and don't run my own with my added routes), i'm not checking that my added routes within the foreman routes are fully 'permissioned'.  Do you have another suggestion to accomplish this?